vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -202,6 +202,7 @@ function Test-ImagesPulled {
         }
     }
     if ($containerRuntime -eq 'containerd') {
+        Start-Job -Name containerd -ScriptBlock { containerd.exe }
```

Comment:
stop and remove containerd is not here, cause this is for testing, and we may have other cases in this cache test requiring container running.

Reply:
Can you help to add above comments in the script?
Does docker test fail because docker is not started before using `docker images`? Should we also start docker for the test?

Reply:
This cannot explain why docker fails occasionally. I think docker starts as a service, so we don't have to start docker every time we start a VM from the OS disk

Reply:
It may explain that it fails occasionally. The docker service is automatically started but it fails when executing `docker images` before it is started. So if we can start docker service manually before executing `docker images`, I guess that we should not hit this issue again.

Reply:
What do you mean by automatically?

Reply:
I have a PR working on the fix of flaky docker vhd cache test https://github.com/Azure/AgentBaker/pull/592.
I enlarged the retry times and still, the test fails for no running docker. Maybe we can check the service list or something in that PR to see if the docker is registered as a service.

Reply:
I think that you can try to add command in #592 to start docker service before calling `docker images` but not retry. This should not block this PR.